README.md

```diff
@@ -44,6 +44,7 @@ Required packages:
 For development and contributions:
 - clang-format-15.0 (can be installed with `python -m pip install clang-format==15.0.7`)
 - cmake-format-0.6 (can be installed with `python -m pip install cmake-format==0.6.13`)
+- black
```

Comment:
black (can be installed with `python -m pip install black==24.3.0`)

Reply:
Done, thanks!